### PR TITLE
Improve basic getting-started docs

### PR DESCRIPTION
**Pull request by papb**
I've reworked the getting-started docs, improving explanations, simplifying some other things, adding more examples and adding more links, hopefully making it easier for new Sequelize users to grasp the basics.

This PR also has some changes for README.md and the index.md from docs, making both more similar to each other. I've also added more nice badges and replaced the badge provider to [badgen.net](https://badgen.net) which I've grown to like a lot - it's fast and reliable.

----------

**NOTE:** I've been wanting to help with a major rework of the sequelize tutorials for a while now, I finally had the time to make some changes, but I will probably be opening more PRs in the future to improve the rest of the tutorials/guides. IMO, they look a bit outdated and disorganized. But I am short on time currently. I will probably suggest some layout changes as well in the future (short future, hopefully).

Thanks for looking! And thanks for the v5 release :grimacing:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,25 +1,24 @@
...
-![node](https://img.shields.io/node/v/sequelize.svg)
-[![License](https://img.shields.io/npm/l/sequelize.svg?maxAge=2592000?style=plastic)](https://github.com/sequelize/sequelize/blob/master/LICENSE)
+[![node](https://badgen.net/npm/node/sequelize)](https://www.npmjs.com/package/sequelize)
+[![License](https://badgen.net/github/license/sequelize/sequelize)](https://github.com/sequelize/sequelize/blob/master/LICENSE)
 [![semantic-release](https://img.shields.io/badge/%20%20%F0%9F%93%A6%F0%9F%9A%80-semantic--release-e10079.svg)](https://github.com/semantic-release/semantic-release)
 
-Sequelize is a promise-based Node.js ORM for Postgres, MySQL, MariaDB, SQLite and Microsoft SQL Server. It features solid transaction support, relations, read replication and more.
+Sequelize is a promise-based Node.js ORM for Postgres, MySQL, MariaDB, SQLite and Microsoft SQL Server. It features solid transaction support, relations, eager and lazy loading, read replication and more.
 
-## v5 Release
```
Comment (sushantdhiman) [inline README.md]: Keep this section for v5

File: README.md [hunk 2]
```diff
@@ -1,25 +1,24 @@
...
-
-You can find upgrade guide and changelog [here](https://github.com/sequelize/sequelize/blob/master/docs/upgrade-to-v5.md)
+Sequelize follows [SEMVER](http://semver.org). Supports Node v6 and above to use ES6 features.
```
Comment (sushantdhiman) [inline README.md]: Move this before `v5` section

File: README.md (hunk 1 above)
Comment (papb) [reply README.md]: Done, let me know if you would like further changes :)

**Comment by papb** [comment]
@sushantdhiman By the way, a meta question: did github automatically ping you because of my comment on your review? Or were you only pinged now, because of this comment which explicitly used @sushantdhiman syntax? Thanks.

**Comment by sushantdhiman** [comment]
@papb I am watching this repository so I get notified for any changes in this repo, that includes all issues and pull requests, but you can ping me if you want to draw my attention to some thread

File: README.md [hunk 3]
```diff
@@ -1,25 +1,26 @@
...
 [![semantic-release](https://img.shields.io/badge/%20%20%F0%9F%93%A6%F0%9F%9A%80-semantic--release-e10079.svg)](https://github.com/semantic-release/semantic-release)
 
-Sequelize is a promise-based Node.js ORM for Postgres, MySQL, MariaDB, SQLite and Microsoft SQL Server. It features solid transaction support, relations, read replication and more.
+Sequelize is a promise-based Node.js ORM for Postgres, MySQL, MariaDB, SQLite and Microsoft SQL Server. It features solid transaction support, relations, eager and lazy loading, read replication and more.
 
-## v5 Release
+Sequelize follows [SEMVER](http://semver.org). Supports Node v6 and above to use ES6 features.
 
-You can find upgrade guide and changelog [here](https://github.com/sequelize/sequelize/blob/master/docs/upgrade-to-v5.md)
+## v5 Release (March 13, 2019)
```
Comment (sushantdhiman) [inline README.md]:
```suggestion
## v5 Release
```

File: README.md [hunk 4]
```diff
@@ -1,25 +1,26 @@
...
 
-```bash
-npm install --save sequelize # will install v5
-```
+You can find the upgrade guide and changelog on [upgrade-to-v5.md](https://github.com/sequelize/sequelize/blob/master/docs/upgrade-to-v5.md).
```
Comment (sushantdhiman) [inline README.md]: English is not my first language but I think in make sense here

```suggestion
You can find the upgrade guide and changelog in [upgrade-to-v5.md](https://github.com/sequelize/sequelize/blob/master/docs/upgrade-to-v5.md).
```

**Pull request merged**